package.json

```diff
@@ -30,8 +30,16 @@
     ]
   },
   "devDependencies": {
+    "@rnw-scripts/beachball-config": "0.0.0",
+    "@rnw-scripts/format-files": "1.0.29",
+    "@rnw-scripts/integrate-rn": "1.1.6",
+    "@rnw-scripts/just-task": "2.2.3",
+    "@rnw-scripts/promote-release": "2.0.4",
+    "@rnw-scripts/stamp-version": "0.0.0",
+    "@rnw-scripts/take-screenshot": "1.0.20",
     "beachball": "^2.20.0",
     "husky": "^4.2.5",
+    "react-native-platform-override": "1.6.6",
     "unbroken": "1.0.27",
     "lage": "^0.29.3"
   },
```

Comment:
This hoists our scripts with `bin` entries, so that they can be run from the repo root, or other projects.